components/cam/bld/namelist_files/namelist_definition.xml

```diff
@@ -5862,62 +5850,130 @@ Default: Set by build-namelist.
 </entry>
 
 <entry id="se_ne" type="integer" category="se"
        group="ctl_nl" valid_values="" >
-Element width of resolution.
-Must match value of grid. Do NOT set this yourself.
+Use internally generated cubed sphere mesh of size 6 x se_ne x se_ne.                         
+If se_ne=0 (RRM grids), read mesh from Exodus file.          
 Default: Set by build-namelist.
 </entry>
 
-<entry id="qsplit" type="integer" category="se"
+<entry id="mesh_file" type="char*256" input_pathname="abs" category="se"
+       group="ctl_nl" valid_values="" >
+Exodus format grid file used when se_ne=0
+Default: Set by build-namelist.
+</entry>
+
+<entry id="se_partmethod" type="integer" category="se"
+       group="ctl_nl" valid_values="4" >
+Mesh partitioning method.  The only supported value in CAM is 4 (space-filling).
+Default: 4
+</entry>
+
+<entry id="statefreq" type="integer" category="se"
+       group="ctl_nl" valid_values="" >
+Frequency with which diagnostic output is written to log (output every
+statefreq dynamics timesteps).
+Default: Set by build-namelist.
+</entry>
+
+<entry id="se_topology" type="char*80" category="se"
+       group="ctl_nl" valid_values="cube" >
+SE's topology.
+Only "cube" is supported in CAM.
+Default: Set by build-namelist.
+</entry>
+
+<entry id="vthreads" type="integer" category="se"
+       group="ctl_nl" valid_values="" >
+Number of vertical threads.
+Default: 1.
+</entry>
+
+<entry id="theta_hydrostatic_mode" type="logical" category="se"
        group="ctl_nl" valid_values="" >
-Tracer advection is done every qsplit dynamics timesteps.
+Run theta model in hydrostatic mode.
+Default: true. 
+</entry>
+
+<entry id="theta_advect_form" type="integer" category="se"
+       group="ctl_nl" valid_values="" >
+Indicates form of eqn. for vtheta in theta model.
+0 is conservation form, 1 is expanded div form.
+Default: 1.
+</entry>
+	
+<entry id="cubed_sphere_map" type="integer" category="se"
+       group="ctl_nl" valid_values="" >
+Defines which map is used for referene element in homme.
+Must be 2 for compsets using TempestRemap mapping files.          
+Default: 0
+</entry>
+
+
+<!-- timestep parameters -->
+
+<entry id="se_nsplit" type="integer" category="se"
+       group="ctl_nl" valid_values="" >
+Number of dynamics steps per physics timestep.
+dt_remap = dtime/se_nsplit                                                                    
+dt_tracer = dt_remap / rsplit                                                                 
+dt_dyn = dt_tracer / qsplit                                                                   
+dt_dyn_vis = dt_dyn / hypervis_subcycle                                                       
+dt_tracer_vis = dt_tracer / hypervis_subcycle_q     
 Default: Set by build-namelist.
 </entry>
 
 <entry id="rsplit" type="integer" category="se"
        group="ctl_nl" valid_values="" >
-Vertically lagrangian code updates every rsplit tracer timesteps.
+Number of tracer steps per remap step.   
```

Comment:
Very minor: is this description correct? With 3D transport, rsplit is more for dynamics, not tracers?

Reply:
I was trying to make the comments consistent with this text:

Number of dynamics steps per physics timestep.                                                                 
dt_remap = dtime/se_nsplit                                                                                     
dt_tracer = dt_remap / rsplit                                                                                  
dt_dyn = dt_tracer / qsplit                                                                                    
dt_dyn_vis = dt_dyn / hypervis_subcycle                                                                        
dt_tracer_vis = dt_tracer / hypervis_subcycle_q        

it's true that rsplit can have a different meaning with 3D transport.  But I think for 3D transport we will always use the new interface, and thus I suggest we dont explain this potential use of rsplit.